yarn.lock

```diff
@@ -13580,3 +13580,4 @@ zod@^3.21.4:
   version "3.23.0"
   resolved "https://registry.yarnpkg.com/zod/-/zod-3.23.0.tgz#a25dab5052702834233e0041e9dd8b6a8480e744"
   integrity sha512-OFLT+LTocvabn6q76BTwVB0hExEBS0IduTr3cqZyMqEDbOnYmcU+y0tUAYbND4uwclpBGi4I4UUBGzylWpjLGA==
+  
```

Comment:
this will cause a diff if someone runs `yarn install` locally

Reply:
Just run it locally and didn't see any diff. 

```
lodestar on  nh/7603-vitest-workspaces [$] via  v22.13.0
❯ yarn
yarn install v1.22.22
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.24s.

lodestar on  nh/7603-vitest-workspaces [$] via  v22.13.0
❯ git status
On branch nh/7603-vitest-workspaces
nothing to commit, working tree clean

lodestar-3 on  nh/7603-vitest-workspaces [$] via  v22.13.0
❯
```